sormas-ui/src/main/java/de/symeda/sormas/ui/externalmessage/processing/AbstractProcessingFlow.java

```diff
@@ -127,6 +131,24 @@ protected CaseDataDto buildCase(PersonDto person, ExternalMessageDto externalMes
 		caseDto.setReportingUser(user.toReference());
 		caseDto.setReportDate(
 			externalMessageDto.getCaseReportDate() != null ? externalMessageDto.getCaseReportDate() : externalMessageDto.getMessageDateTime());
+
+		FacilityReferenceDto personFacility = externalMessageDto.getPersonFacility();
+		if (personFacility != null && FacadeProvider.getConfigFacade().isConfiguredCountry(CountryHelper.COUNTRY_CODE_LUXEMBOURG)) {
+			FacilityDto facility = FacadeProvider.getFacilityFacade().getByUuid(personFacility.getUuid());
+			FacilityType facilityType = facility.getType();
+
+			caseDto.setResponsibleRegion(facility.getRegion());
+			caseDto.setResponsibleDistrict(facility.getDistrict());
+
+			if (facilityType.isAccommodation()) {
+				caseDto.setFacilityType(facilityType);
+				caseDto.setHealthFacility(personFacility);
+			} else {
+				FacilityDto noneFacility = FacadeProvider.getFacilityFacade().getByUuid(FacilityDto.NONE_FACILITY_UUID);
```

Comment:
Unnecessary variable declaration, it's only used once and doesn't improve readability